assignment-1/src/app/chat-bubble/chat-bubble.component.html

```diff
@@ -0,0 +1,8 @@
+<div class="chat-bubble" [innerHTML]="singleMessage.message" 
+[ngClass]="{'right-chat-bubble':getBubbleSide()=='right','left-chat-bubble':getBubbleSide()=='left'}"
+[ngStyle]="{backgroundColor:getBubbleSide()=='right' ? getBubbleColor():'',borderRadius:getBubbleContainerSide()}"
```

Comment:
nit: modify styles only with external styles, handle style changes with classes using ngClass and apply styles in the classes to improve code readability

Reply:
External styles where handled in css instead for writing the logic in type script.